src/containers/shared/components/Transaction/EscrowCancel/parser.ts

```diff
@@ -0,0 +1,25 @@
+import formatAmount from '../../../../../rippled/lib/txSummary/formatAmount'
+
+const findNode = (meta: any) => {
```

Comment:
Could this use `import { findNode } from '../../../transactionUtils'` instead?

Reply:
I think this findNode is escrow specific... not sure though
this is the findNode from TransactionUtils:
export function findNode(
  transaction: Transaction,
  nodeType: keyof Node,
  entryType: string,
): Node {
  const metaNode = transaction.meta.AffectedNodes.find(
    (node) => node[nodeType] && node[nodeType].LedgerEntryType === entryType,
  )
  return metaNode ? metaNode[nodeType] : null
}

Reply:
I was looking at how it is used in Description.tsx which I think is doing the same thing.

```
const deleted = findNode(data.tx, 'DeletedNode', 'Escrow')
```